WTWSMS/history/provinces/478 - Monemvasia.txt

```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
```

Comment:
Prosperity is a system which adds holdings. This place was neither fertile nor populous historically in the timeframe. But if you change that you can get 6 holdings.